tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -145,6 +146,21 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
         }
     }
 
+    // Remove IngressRouteTCP route for stopped instances
+    let ingress_route_tcp_name = format!("{}-ro-0", cdb.name_any().as_str());
```

Comment:
Where are you getting this name from?  The names would need to be looked up somehow.

For example the `IngressRouteTCP` name on most of them end with `rw-0`, but we will also need to handle any custom domain configurations as well.